study-builder/studies/rgp/study.conf

```diff
@@ -0,0 +1,171 @@
+{
+  "tenant": {
+    "domain": ${auth0.domain},
+    "mgmtClientId": ${auth0.mgmtClientId},
+    "mgmtSecret": ${auth0.mgmtSecret}
+  },
+
+  "umbrella": {
+    "name": "RGP",
+    "guid": "rgp"
+  },
+
+  "study": {
+    "name": "rgpproject",
+    "guid": "RGP",
+    "studyEmail": "rgp@dev.datadonationplatform.org",
+    "baseWebUrl": ${baseWebUrl},
+    "irbPassword": ${irbPassword},
+    "plusCodePrecision": MEDIUM,
+    "shareParticipantLocation": true
```

Comment:
I recommend setting this to `false`, unless we know for sure that the RGP study wants to share the location pluscodes of their participants.

Reply:
Good idea.  I think I copied that from another study config file